lmp/bb-build.sh

```diff
@@ -18,4 +18,4 @@ bitbake -e ${IMAGE} > ${archive}/bitbake_image_env.txt
 # Setscene (cache), failures not critical
 bitbake --setscene-only ${IMAGE} || true
 
-bitbake -D ${IMAGE}
+bitbake ${BITBAKE_ARGS} ${IMAGE}
```

Comment:
removing -D by default will impact in the debug artifacts we provide on each factory. I don't know if I like it, but I don't have hard feelings, so I'm ok with the change.

However, I would prefer to have `BITBAKE_ARGS` to have `-D` as initial default, at least.

Reply:
@angolini Line 26 in `lmp/bb-config.sh` does exactly what you ask.

Reply:
Right, but do we want to disable -D here? I know this is an advanced customization, but I wonder if it would be better to force -D instead and allow extra arguments via BITBAKE_ARGS.

@quaresmajose how were you planning on using this?

Reply:
Yup, it is fine to have it as default when not set, I'm just wondering if there is any bad side effect of forcing -D on all cases (and allowing additional arguments).

Reply:
I think no, it can work like you said: `-D` forced and `BITBAKE_ARGS` to extend

Reply:
Cool, then lets keep -D and just allow extra args instead (that will help keeping the builds somehow behaving similarly).